src/Arcus.Security.Tests.Integration/KeyVault/KeyVaultSecretProviderTests.cs

```diff
@@ -86,6 +132,26 @@ public async Task KeyVaultSecretProvider_WithCustomManagedServiceIdentity_GetSec
             Assert.NotNull(secret.Version);
         }
 
+        [Fact]
+        public async Task KeyVaultSecretProvider_WithCustomManagedIdentity_GetSecret_Succeeds()
```

Comment:
What od you mean with Custom Managed Identity? Do you mean User-assigned?

Reply:
Yeah, meaning that we created a managed identity ourselves and didn't use the system identities.

Reply:
Ok let's rename it to UserAssignedManagedIdentity then, that's how Azure calls it.

Reply:
Done, resolving.